packages/main/src/ModuleRunner.ts

```
@@ -9,9 +9,18 @@ class ModuleRunner implements PromiseLike<void> {
     this.#promise = Promise.resolve();
   }
 
-  then<TResult1 = void, TResult2 = never>(onfulfilled?: ((value: void) => TResult1 | PromiseLike<TResult1>) | null | undefined, onrejected?: ((reason: any) => TResult2 | PromiseLike<TResult2>) | null | undefined): PromiseLike<TResult1 | TResult2> {
-        return this.#promise.then(onfulfilled, onrejected);
-    }
+  then<TResult1 = void, TResult2 = never>(
```

Biome (2.1.2)

[error] Do not add then to a class.



(lint/suspicious/noThenProperty)